EFCore.InMemory.HierarchyId/EFCore.InMemory.HierarchyId.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <AssemblyName>EntityFrameworkCore.InMemory.HierarchyId</AssemblyName>
+    <RootNamespace>Microsoft.EntityFrameworkCore.InMemory</RootNamespace>
+    <Authors>Brice Lambson, Adrian Paul Nutiu, et al.</Authors>
+    <Description>Adds hierarchyid support to the in-memory EF Core provider</Description>
+    <PackageProjectUrl>https://github.com/efcore/EFCore.SqlServer.HierarchyId</PackageProjectUrl>
+    <PackageTags>EFCore;SQL Server;HierarchyId</PackageTags>
```

Comment:
May be drop SqlServer and add InMemory?

Reply:
It slipped! Updated in https://github.com/efcore/EFCore.SqlServer.HierarchyId/pull/33/commits/22b1e03885e4b79a0bb762cfa6e0bf33b86927dd